package.json

```diff
@@ -49,7 +49,8 @@
     "pnpm": {
         "overrides": {
             "ansi-regex": "^5.0.1",
-            "tmpl": "^1.0.5"
+            "tmpl": "^1.0.5",
+            "vm2": ">=3.9.6"
```

Comment:
I believe that ^ is the more appropriate form for the overrides https://pnpm.io/package_json#pnpmoverrides
Note also that there are a number of breaking changes, issues and performance issues fixed in these versions 3.9.6 to 3.9.9.

Reply:
Thanks for the link! Based on it, I would go even further and use `"vm2@<3.9.6": "^3.9.6"`.